database/generate.js

```diff
@@ -1,27 +1,27 @@
-var faker = require('faker');
-var fs = require('fs');
+let faker = require('faker');
+let fs = require('fs');
 
-var sizeArray = ['XXS', 'XS', 'S', 'M', 'L', 'XL', 'XXL'];
-var heightArray = ['Under 4\'8', '4\'10', '5\'0', '5\'2', '5\'4', '5\'6', '5\'8', '5\'10', '6\'0', '6\'2', 'Over 6\'2'];
-var ageArray = ['17 or under', '18 to 24', '25 to 34', '35 to 44,', '45 to 54', '55 to 64', '65 or over'];
-var genderArray = ['M', 'F'];
+let sizeArray = ['XXS', 'XS', 'S', 'M', 'L', 'XL', 'XXL'];
+let heightArray = ['Under 4\'8', '4\'10', '5\'0', '5\'2', '5\'4', '5\'6', '5\'8', '5\'10', '6\'0', '6\'2', 'Over 6\'2'];
+let ageArray = ['17 or under', '18 to 24', '25 to 34', '35 to 44,', '45 to 54', '55 to 64', '65 or over'];
+let genderArray = ['M', 'F'];
 
-var data = [];
-for (var i = 0; i < 100; i++) {
-    for (var j = 0; j < Math.floor(Math.random() * 5) + 3; j++) {
+let data = [];
+for (let i = 0; i < 100; i++) {
+    for (let j = 0; j < Math.floor(Math.random() * 5) + 3; j++) {
 
-        var pickAge = Math.floor(Math.random() * ageArray.length);
-        var pickHeight = Math.floor(Math.random() * heightArray.length);
-        var pickSize = Math.floor(Math.random() * sizeArray.length);
-        var pickGender = Math.floor(Math.random() * genderArray.length);
-        var chestSize = Math.floor(Math.random() * (52 - 30 + 1)) + 30;
-        var waistSize = Math.floor(Math.random() * (35 - 28 + 1)) + 28;
-        var hipsSize = Math.floor(Math.random() * (35 - 28 + 1)) + 28;
-        var fitRating = Math.floor(Math.random() * 5) + 1;
-        var starRating = Math.floor(Math.random() * 5) + 1;
-        var lengthRating = Math.floor(Math.random() * 5) + 1;
-        var qualityRating = Math.floor(Math.random() * 5) + 1;
-        var helpfulRating = Math.floor(Math.random() * 15);
+        let pickAge = Math.floor(Math.random() * ageArray.length);
```

Comment:
Problem(s): If someone wants to adjust the min/max for any of these values, it is non-obvious how to go about doing that. If you decide to use a different random number generation scheme (say you want a non-uniform distribution, for example), then there is a ton of code to change here.

Fix: Abstract this code into a `randomIntegerBetween(min, max)` function. 

Note: I also don't think saving these values into variables is helping readability. Yes, they are now named, but, I think they can be intuited from the object keys on the following lines. If you created the function above (as well as a similar function, `chooseRandomElement(array)`), then the part where you construct the object literal starting on line 27 on could look something like 

```
data.push({
  waist_size: randomIntegerBetween(28, 35),
  height: choseRandomElement(heightArray),
  // etc
})
```